04/app.js

```diff
@@ -2,8 +2,36 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
-        year: '1985'
+        day: '08',
+        month: '01',
+        year: '1969'
     }
+}
+
+const now = new Date();
+
+const date = [now.getDate(), now.getMonth()]
+  .map(function (value) {
+    if (value === 0) {
+      value++;
+    }
+    if (value < 10) {
+      return "0" + value;
+    }
+  });
```

Comment:
Powiedziałbym, że jest to mało czytelne - oczywiście działa, ale nie kombinowałbym aż tak, aby rozwiązać taki problem. Napisałbym funkcję zerofill (https://stackoverflow.com/a/20460414) lub podobną, która dodaje 0 na początek jak trzeba, a poza tym po prostu pobrał dzień i miesiąc osobno (do którego dodał po prostu 1)